tilewe/engine.py

```diff
@@ -113,24 +119,30 @@ class LargestPieceEngine(Engine):
     ties, it's effectively a greedy form of RandomEngine.
     """
 
-    def __init__(self, name: str="LargestPiece", estimated_elo: float=None):
-        super().__init__(name, 30.0 if estimated_elo is None else estimated_elo)
+    def __init__(self, name: str="LargestPiece", style: str="max", estimated_elo: float=None):
```

Comment:
Should the name still default to "LargestPiece" even when the style is set to "min"? I'm not sure what the most elegant way to fix it is, maybe we could make `name` default to none and set it based on style (since it's a required parameter) if the user didn't pass a value for it. Open to other ideas too